diff --git a/src/core/Config.cpp b/src/core/Config.cpp
index ed570cd..b819c7d 100644
--- a/src/core/Config.cpp
+++ b/src/core/Config.cpp
@@ -98,9 +98,7 @@ static const QHash<Config::ConfigKey, ConfigDirective> configStrings = {
     {Config::GUI_HidePreviewPanel, {QS("GUI/HidePreviewPanel"), Roaming, false}},
     {Config::GUI_AlwaysOnTop, {QS("GUI/GUI_AlwaysOnTop"), Local, false}},
     {Config::GUI_ToolButtonStyle, {QS("GUI/ToolButtonStyle"), Roaming, Qt::ToolButtonIconOnly}},
-#ifdef KEEPASSXC_DIST_FLATPAK
     {Config::GUI_LaunchAtStartup, {QS("GUI/LaunchAtStartup"), Roaming, false}},
-#endif
     {Config::GUI_ShowTrayIcon, {QS("GUI/ShowTrayIcon"), Roaming, false}},
     {Config::GUI_TrayIconAppearance, {QS("GUI/TrayIconAppearance"), Roaming, {}}},
     {Config::GUI_MinimizeToTray, {QS("GUI/MinimizeToTray"), Roaming, false}},
diff --git a/src/core/Config.h b/src/core/Config.h
index 9624a1a..b347ea7 100644
--- a/src/core/Config.h
+++ b/src/core/Config.h
@@ -79,9 +79,7 @@ public:
         GUI_HidePreviewPanel,
         GUI_AlwaysOnTop,
         GUI_ToolButtonStyle,
-#ifdef KEEPASSXC_DIST_FLATPAK
         GUI_LaunchAtStartup,
-#endif
         GUI_ShowTrayIcon,
         GUI_TrayIconAppearance,
         GUI_MinimizeToTray,
